src/resources/elements/editingCard/editingCard.ts

```diff
@@ -0,0 +1,24 @@
+import { bindable } from "aurelia-framework";
+import "./editingCard.scss";
+
+export class EditingCard {
```

Comment:
@dkent600 @IonelLupu 

I created a separate "<editing-card>" component.
Reason:
- Ionel used that in Stage5 (we could refactor)
- Because I basically copy pasted Ionel's card logic, I felt like, I did not want to "pollute" Terms Clauses code
- --> Separate `<editing-card>` component

Reply:
I think there is a small problem with the Save functionality. What happens if my Save button needs to do something but then, due to some login in my Save method, I don't want to switch to the "view" mode. For example, if my validation fails.

Maybe we can make the onSave method return a boolean. If the return value is true, then we switch to the "view" mode. Other suggestions are welcomed